chapters/queues.md

```diff
@@ -0,0 +1,100 @@
+# Queues
+
+> Further resources for queues from [AMD](https://gpuopen.com/learn/concurrent-execution-asynchronous-queues/) and [NVIDIA](https://www.khronos.org/assets/uploads/developers/library/2016-vulkan-devday-uk/9-Asynchonous-compute.pdf)
+
+A `VkQueue` is what an application submits work to, normally in the form of `VkCommandBuffer` objects or [sparse bindings](./sparse_resources.md).
```

Comment:
I know we're trying to define VkQueue, but allowing for highlighting, I hope "An application submits work to a 'VkQueue'..." would be clear enough without the awkward wording?

Otherwise "A 'VkQueue' is the type of object to which an application submits work", but this is the type of English with which we will not put...